package.json

```diff
@@ -71,5 +71,37 @@
     "aws-cdk": "2.157.0",
     "aws-cdk-lib": "2.157.0",
     "constructs": "10.3.0"
+  },
+  "exports": {
+    ".": "./lib/index.js",
+    "./lib/constants/access": "./lib/constants/access.js",
+    "./lib/constants/context-keys": "./lib/constants/context-keys.js",
+    "./lib/constants/fastly-aws-account-ids": "./lib/constants/fastly-aws-account-ids.js",
+    "./lib/constants/library-info": "./lib/constants/library-info.js",
+    "./lib/constants/metadata-keys": "./lib/constants/metadata-keys.js",
+    "./lib/constants/regex-pattern": "./lib/constants/regex-pattern.js",
+    "./lib/constants/ssm-parameter-paths": "./lib/constants/ssm-parameter-paths.js",
+    "./lib/constants/tracking-tag": "./lib/constants/tracking-tag.js",
```

Comment:
I'm not supeerrr happy with the lack of consistency in our modules, sometimes a single file is a module like our constants, or `root.ts` and sometimes we treat an entire folder as a module.

I'd like to avoid a big refactor for the time being so I'll just make it work, but might be worth exploring in a future cleanup.